test-requirements.txt

```diff
@@ -1,8 +1,10 @@
+ansible
```

Comment:
please drop this from requirements file. We don't actually need ansible, as we set it up by default.

Reply:
Fixed